### DISCUSSION TITLE
Can’t start on Synology

### DISCUSSION THREAD

**Discussion by Gitcrt5** [haugene/docker-transmission-openvpn#2752] [Q&A]
I have just tried installing this on a Synology DS918+.  I am no expert so just tried to do the basics. My Docker is new. It is the only package. I downloaded this package and set it up to use PIA. When I start it I get an error in the log :
`Options error: the --route-pre-down directive should have at most 1 parameter`
Then it stops. I have followed the steps listed for trouble shooting at https://haugene.github.io/docker-transmission-openvpn/debug/ and they all work. I'm not sure what I need to do or edit.

**Comment by pkishino** [comment]
Post the whole log please

**Comment by 2b-arc** [comment]
I have the same issue. 
[haugene-transmission-openvpn-2.csv](https://github.com/haugene/docker-transmission-openvpn/files/13604597/haugene-transmission-openvpn-2.csv)

**Reply by pkishino** [reply]
your using nordvpn and don't look like you setup config properly, check the documentation please...

**Reply by wildoracle** [reply]
I'm having the same problem as well, and this is so frustrating because I had similar issues when first setting this up years ago. It had been running just fine, until Synology Container Manager told me I needed to update the container. After doing that, it won't start again. I should have followed the old adage, "if it ain't broke don't fix it". 
Now the container starts and then stops. I had to go through hoops and hurdles back when I set it up, following [this guide](https://www.reddit.com/r/VPNTorrents/comments/900w78/xpost_rsynology_setting_up_the/) to create a TUN.sh startup script, and selecting the "execute container using high privilege" in order to get things working 
I've tried everything I can find. Attached is my log file.
[haugene-transmission-openvpn.csv](https://github.com/haugene/docker-transmission-openvpn/files/13927450/haugene-transmission-openvpn.csv)
THANK YOU!

**Comment by Gitcrt5** [comment]
Here is the log.
[haugene-transmission-openvpn1.csv](https://github.com/haugene/docker-transmission-openvpn/files/13604981/haugene-transmission-openvpn1.csv)

**Reply by pkishino** [reply]
Please also post your config, seems like the error is in the config

**Reply by Gitcrt5** [reply]
I didn't use a configuration file, I set it up in the docker package on technology by putting in the details in the docker GUI environment section. I found these files in the docker subdirectory, is this what you mean?

**Reply by pkishino** [reply]
you just uploaded your vpn credentials..I deleted the post but you best change them anyway

**Comment by Gitcrt5** [comment]
![docker](https://github.com/haugene/docker-transmission-openvpn/assets/153168097/96058ba0-1573-49e4-bde9-3cfd15829494)

**Reply by pkishino** [reply]
yes, that is the config..but I would need the see the whole one please

**Reply by Gitcrt5** [reply]
How do I export it out of the Synology Docker?

**Reply by Gitcrt5** [reply]
Was that it? I think I found where to do it.

**Reply by pkishino** [reply]
yes..but again, please be more careful with sharing your credentials..

**Reply by pkishino** [reply]
yes, your config is bad..
OPENVPN_OPTS you have modified, you should specify the config in OPENVPN_CONFIG

**Reply by pkishino** [reply]
like this
-e OPENVPN_PROVIDER=PIA \
              -e OPENVPN_CONFIG=france \
              -e OPENVPN_USERNAME=user \
              -e OPENVPN_PASSWORD=pass \

**Reply by pkishino** [reply]
https://haugene.github.io/docker-transmission-openvpn/config-options/

**Reply by Gitcrt5** [reply]
If you have a look at my screenshot there is no OPENVPN_CONFIG field to use. I only filled in the options that were available to me.

**Reply by Gitcrt5** [reply]
Ahh. I see. You can edit those options. Sorry, I did not know that. It was presented as an option to fill out and I did not think to edit it. I'll try that now.

**Comment by pkishino** [comment]
Please look at the documentation,
The dsm gui is a terrible tool for this and you most likely need to create
the field yourself or rename the openvpn_opts field

**Reply by Gitcrt5** [reply]
Thanks. Not being an advanced user I just went with the easiest way forward for me. I thought of using a config file but it wasn't clear to me where I should put it. And if I did want to use a config file how I would tell Docker to use that file rather than the options from the GUI.

**Comment by pkishino** [comment]
please look at dsm forums or support for dsm as that docker package is
built for dsm, I do not know

**Reply by Gitcrt5** [reply]
Tried again. Still failed. Logs etc included.
[haugene-transmission-openvpn1 (1).json](https://github.com/haugene/docker-transmission-openvpn/files/13609132/haugene-transmission-openvpn1.1.json)
[haugene-transmission-openvpn1.csv](https://github.com/haugene/docker-transmission-openvpn/files/13609133/haugene-transmission-openvpn1.csv)

**Reply by pkishino** [reply]
different error:
ERROR: Cannot ioctl TUNSETIFF tun: Operation not permitted (errno=1)

please check on dsm forums or such, this is a permission problem..

**Reply by Gitcrt5** [reply]
OK. Thanks for all of your help.

**Comment by metaclam** [comment]
I'm in a [very similar boat](https://github.com/haugene/docker-transmission-openvpn/discussions/2838) Gitcrt5.   I agree that using the Container Manager GUI would be far simpler to use, but why does it populate with irrelevant or incorrect "deafult" keys?  That's coming from where?

Anyway did you get any answers on any dsm forums?

**Comment by borland502** [comment]
If anyone finds this thread and had to follow the faq to avoid a soft reset and is using a docker-compose:

```
- OPENVPN_OPTS=--inactive 3600 --ping 10 --ping-exit 60 --pull-filter ignore ping
```

should not have any quotes, contrary to the container log's advice
